### PR TITLE
cleanup `if statements` in `property_get`

### PR DESCRIPTION
**Pull request by bekzod**
gave little improvement in benchmarks https://cl.ly/0V1f1g19103B/results.pdf

### CODE REVIEW

**Comment by krisselden** [comment]
This is out of date.

File: packages/ember-metal/lib/property_get.js [hunk 1]
```diff
@@ -95,56 +95,51 @@ export function get(obj, keyName) {
...
+  if (descriptor === undefined) {
+    if (DEBUG && HAS_NATIVE_PROXY) {
+      value = getPossibleMandatoryProxyValue(obj, keyName);
+    } else {
+      value = obj[keyName];
     }
 
-    if (descriptor !== undefined) {
-      return descriptor.get(obj, keyName);
+    if (isDescriptor(value)) {
```
Comment (krisselden) [inline packages/ember-metal/lib/property_get.js]: it is weird to do this call, when at this point descriptor could be defined and value === undefined

Comment (bekzod) [reply packages/ember-metal/lib/property_get.js]: maybe add test case what you mean or suggest change :P ? I just reorganized `if`s but the logic is same as was before

**Comment by bekzod** [comment]
I think this is out of date

**Pull request closed without merging**